### PR TITLE
fix log trap 178182355

### PR DESCRIPTION
**Pull request by zack-erisyon**
- wip
- wip
- tests passing

### CODE REVIEW

File: plaster/run/run_main.py [hunk 1]
```diff
@@ -79,123 +80,128 @@ def main(self, job_folder=None):
...
         # At this point job_folder is a plumbub path
 
-        dup_log_path = job_folder / f"{int(time.time()):06x}.log"
-        with zlog.log_trap(
-            "file", log_name=None, file_path=dup_log_path, formatter_name="json"
-        ):
-            tell(f"Trapping run logs into {dup_log_path}")
+        # Add a new handler so we get PER-RUN log files into the job folder
+        per_run_log_path = job_folder / f"{int(time.time()):06x}.log"
+        formatter = zlog.TypeAwareJsonFormatter(
```
Comment (danwinkler) [inline plaster/run/run_main.py:85]: Does this mean that the log files will be json encoded? They need to be human readable.

File: plaster/run/sigproc_v2/zests/zest_align_ims.py [hunk 2]
```diff
@@ -113,25 +114,27 @@ def _synth(im_mea, n_cycles):
...
-
-                    with prof("resample"):
-                        resample_aligned_ims(chcy_ims, pred_chcy_offsets)
-
-                profile_records = profile_from_string(prof_logs.getvalue())
-                profile_dump(profile_records)
+        # zest.stack_mock(zlog._app_start_print)
+        # zest.stack_mock(profile._profile_dump_print)
+
+        profile.prof_start_record_capture()
```
Comment (danwinkler) [inline plaster/run/sigproc_v2/zests/zest_align_ims.py:120]: The other way you could have implemented this was as a class, such that you could do something like:
```python
align_prof = Profile(**options)
with align_prof:
    # Do work
print(align_prof.elapsed)

# Or in the case where you just want to print the results to stdout
with Profile(**options):
    # Do work
````

**Review by danwinkler — APPROVED** [review]
LGTM, but check the comment about the format of the log file

File: plaster/run/run_main.py (hunk 1 above)
Comment (zack-erisyon) [reply plaster/run/run_main.py:85]: This is really for debugging. You think these extra log files need to be human readable?

Comment (danwinkler) [reply plaster/run/run_main.py:85]: Yes Angela looks at these files regularly

**Pull request merged**